Sources/SwiftDocCUtilities/Action/Actions/IndexAction.swift

```diff
@@ -18,8 +18,6 @@ public struct IndexAction: AsyncAction {
     let bundleIdentifier: String
 
     var diagnosticEngine: DiagnosticEngine
-    
-    private var dataProvider: LocalFileSystemDataProvider!
```

Comment:
This private variable was only set and never read so it could be changed to a locally scoped variable inside `private func buildIndex()` to avoid holding on to the data provider after the action finished performing its action.